src/JS/BigInt.js

```diff
@@ -35,11 +35,11 @@ export const fromStringAsImpl = function (just) {
         while (i < value.length) parts.push(value.slice(i, i += size));
 
         function f(acc, chunk) {
-          let n = BigInt(parseInt(chunk, radix));
+          let n = parseInt(chunk, radix);
           if (isNaN(n)) {
             throw new Error("Invalid number");
           } else {
-            return acc * factor + n;
```

Comment:
Now the question becomes, does this computation do what we expect it to?

Reply:
If I understand the algorithm correctly, the functionality is not altered by this change : the idea, here, seems to use all the work done by parseInt which unfortunately only fits numbers less than MAX_INT. The initial value is cut in chunks, each chunk is converted in BigInt, then all the converted numbers are folded together by function f. My modification only post-converts rather than pre-converts.